testing/regress/ecl/memcachedtest.ecl

```diff
@@ -17,6 +17,7 @@
 
 //class=embedded
 //class=3rdparty
+//nothor
```

Comment:
No they all worked. I agree and had also thought this but didn't thnk it was worth adding a whole new reg test for exceptions, like I do for redis, for just this one test. Such a set of tests should be there but I guillotined my time and would only have felt comfortable adding this test to a sep ecl file if I also took the time to write others in, and have it complete. I will open a jira to do so. I was unaware of the above code, thanks.